src/squidpy/gr/_sepal.py

```diff
@@ -290,8 +281,11 @@ def _entropy(
     """Get entropy of an array."""
```

Comment:
This is what that does right?

```suggestion
    """Compute Shannon entropy of an array of probability values (in nats)."""
```

Nats instead of Shannons, because you use the natural logarithm instead of log2.

Are you sure that `0` is the accurate result if the probability sum is 0?

Shouldn’t it be `np.log(len(xx))`?